### ISSUE TITLE
[bug] Dependency `stream-browserify` is not installed (Solana Pay PoS)

### ISSUE THREAD

**Issue by dineshpinto** [solana-labs/solana-pay#44]
# Expected Behavior
`yarn install` installs all dependencies

# Current Behavior
`yarn install` does not install dependency `stream-browserify`

# Possible Solution
This is easy enough to fix by running `npm install stream-browserify` before running `yarn start`. But maybe it should be added to the dependency list.

# Steps to Reproduce
1. Clean installation of `yarn/npm` on MacOS Monterey 12.2
2. Follow setup instructions as defined in https://github.com/solana-labs/solana-pay/tree/master/point-of-sale `README.md` (commit `ebe5ad9dce68dfcafd426be02ebe2e34f90cd056`)

**Comment by jordansexton** [comment]
Thanks for the report. I've seen this too, I'll figure out which package it's coming from and add it as a dependency if needed.

**Comment by jordansexton** [comment]
## Steps to Replicate

### Install
```
git clone https://github.com/solana-labs/solana-pay.git
cd solana-pay/point-of-sale
yarn
```

Output:
```
yarn install v1.22.17
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
warning Pattern ["buffer@npm:buffer@^6.0.3"] is trying to unpack in the same destination "/Users/jordan/Library/Caches/Yarn/v6/npm-buffer-6.0.3-2ace578459cc8fbe2a70aaa8f52ee63b6a74c6c6-integrity/node_modules/buffer" as pattern ["buffer@~6.0.3","buffer@~6.0.3","buffer@^6.0.3"]. This could result in non-deterministic behavior, skipping.
warning Pattern ["stream-browserify@^3.0.0"] is trying to unpack in the same destination "/Users/jordan/Library/Caches/Yarn/v6/npm-stream-3.0.0-22b0a2850cdf6503e73085da1fc7b7d0c2122f2f-integrity/node_modules/stream" as pattern ["stream@npm:stream-browserify@^3.0.0"]. This could result in non-deterministic behavior, skipping.
```

### Run

```
yarn start
```

Output:
```
yarn run v1.22.17
$ yarn clean && parcel src/index.html
$ shx rm -rf dist .parcel-cache
Server running at http://localhost:1234
🚨 Build failed.

@parcel/core: Failed to resolve 'src/index.html'

@parcel/resolver-default: Cannot find module 'stream-browserify'
Require stack:
- /Users/jordan/Github/solana-pay/point-of-sale/node_modules/@parcel/node-libs-browser/index.js
- /Users/jordan/Github/solana-pay/point-of-sale/node_modules/@parcel/node-resolver-core/lib/builtins.js
- /Users/jordan/Github/solana-pay/point-of-sale/node_modules/@parcel/node-resolver-core/lib/NodeResolver.js
- /Users/jordan/Github/solana-pay/point-of-sale/node_modules/@parcel/resolver-default/lib/DefaultResolver.js
- /Users/jordan/Github/solana-pay/point-of-sale/node_modules/@parcel/package-manager/lib/index.js
- /Users/jordan/Github/solana-pay/point-of-sale/node_modules/@parcel/core/lib/resolveOptions.js
- /Users/jordan/Github/solana-pay/point-of-sale/node_modules/@parcel/core/lib/Parcel.js
- /Users/jordan/Github/solana-pay/point-of-sale/node_modules/@parcel/core/lib/index.js
- /Users/jordan/Github/solana-pay/point-of-sale/node_modules/parcel/lib/cli.js
- /Users/jordan/Github/solana-pay/point-of-sale/node_modules/parcel/lib/bin.js

  Error: Cannot find module 'stream-browserify'
```

cc @steveluscher do you think this could be related to wallet-adapter?

**Comment by steveluscher** [comment]
> …do you think this could be related to wallet-adapter?

Definitely; oof.

1. `wallet-adapter-torus` now aliases `stream` to `npm:stream-browserify@^3.0.0`
2. Parcel also makes use of `stream-browserify` through `node-libs-browser`
3. Yarn goes ‘oh look; stream-browserify is already installed by `wallet-adapter-torus`. Nothing to do here!’
4. When `node-libs-browser` goes to import `stream-browserify` it can't find it. The only path on disk now is `node_modules/stream` because of the alias.

Looking into what we can do about this now.

**Comment by steveluscher** [comment]
OK, yeah, I think I've got it. Hold please
![iu](https://user-images.githubusercontent.com/13243/152284863-c5d03ff0-df93-4db5-881d-acd50701256a.gif)
.

**Comment by itoonx** [comment]
I got the same issue

node version: ```v16.13.2 ```
os: macos chip m1

**Comment by jordansexton** [comment]
Published:
- @solana/pay@0.1.0-alpha.8

This is fixed in the Point of Sale app via a6a83c1c4897624d7dacfe5a2e725585d660faef

Please run `yarn nuke && yarn && yarn start` from the `point-of-sale` directory to wipe the node_modules and run it!

**Comment by jordansexton** [comment]
Thank you @steveluscher for the fix!
